modules/prokka/prokka.nf

```diff
@@ -1,19 +1,44 @@
+nextflow.enable.dsl = 2
 
 process PROKKA {
     tag { pair_id }
     publishDir "${params.output_dir}/prokka", mode: 'copy'
 
     input:
-    tuple val(pair_id), path("${pair_id}.contigs.fa")
+    tuple val(pair_id), path(contigs_file)
+    val(stain_genus_species)
```

Comment:
@emilio-r , this is an example of how we consume the `stdout` of another process. In DSL2 we need to specify the types of value.

`val` is type used for primitive values like `int` or `string` etc.